src/components/header/headerNavbar.tsx

```diff
@@ -63,78 +65,108 @@ const HeaderNavbar = () => {
           <Navbar.Toggle
             aria-controls="basic-navbar-nav"
             className="border border-2 border-tertiary text-white mb-1 w-100 w-sm-auto"
+            onClick={() => handleShowOffCanvas()}
           />
           <Navbar.Offcanvas
             id="responsive-navbar-nav"
-            className="justify-content-center"
+            className="justify-content-center bg-primary border-0"
             placement="end"
+            show={showOffCanvas}
+            onHide={handleCloseOffCanvas}
           >
-            <Row className="justify-content-between">
-              <Col xs={12} lg={9}>
-                <Nav
-                  className="justify-content-center"
-                  style={{ height: "36px", whiteSpace: "nowrap" }}
-                >
-                  <NavLink
-                    to="/"
-                    end={true}
-                    className={({ isActive }) =>
-                      isActive ? activePageStyle : inactivePageStyle
-                    }
-                  >
-                    Home
-                  </NavLink>
-                  <NavLink
-                    to="/browse"
-                    className={({ isActive }) =>
-                      isActive ? activePageStyle : inactivePageStyle
-                    }
+            <Offcanvas.Header
+              closeButton
+              closeVariant="white"
+              className="text-white"
+            >
+              Navigation
+            </Offcanvas.Header>
+            <Offcanvas.Body>
+              <Row className="justify-content-between w-100 mx-0">
+                <Col xs={12} lg={9} className="mb-2 mb-sm-0 px-0">
```

Comment:
```suggestion
                <Col
                  className="mb-2 mb-sm-0 px-0"
                  xs={12}
                  lg={showOffCanvas ? 12 : 9}
                >
```

Can have maximum width if shown off canvas.